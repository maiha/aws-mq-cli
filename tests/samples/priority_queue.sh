#!/bin/bash

aws-mq-cli samples priority_queue run

