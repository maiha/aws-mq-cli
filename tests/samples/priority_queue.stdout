 [x] Published messages with priorities.
 [*] Consuming... (To exit press CTRL+C)
 [x] Received [2] (expected)
 [x] Received [1] (expected)
